acceptance/openstack/compute/v2/servers_test.go

```diff
@@ -276,24 +261,23 @@ func TestActionReboot(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	defer servers.Delete(client, server.ID)
 
 	if err = waitForStatus(client, server, "ACTIVE"); err != nil {
 		t.Fatal(err)
 	}
+	defer func() {
+		servers.Delete(client, server.ID)
+		t.Logf("Server deleted.")
+	}()
 
 	rebootOpts := &servers.RebootOpts{
 		Type: servers.SoftReboot,
 	}
-	res := servers.Reboot(client, server.ID, rebootOpts)
-	if res.Err == nil {
-		t.Fatal("Expected the SDK to provide an ArgumentError here")
-	}
```

Comment:
I wasn't sure why this was in the test. Rather than commenting it out, I figured I would yank it out, comment here, and add it back if necessary. :)